src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/SmokeTestHelperResourceIntTest.java

```diff
@@ -81,5 +88,59 @@ public void willAddImprisonmentStatus() {
         assertThat(statuses.get(1).isActiveLatestStatus()).isTrue();
     }
 
+    @Nested
+    public class ReleasePrisoner {
+
+        @Test
+        @DisplayName("requires ROLE_SMOKE_TEST")
+        public void requiresCorrectRole() {
+            final var request = createHttpEntity(authTokenHelper.getToken(AuthTokenHelper.AuthToken.SYSTEM_USER_READ_WRITE), null);
+
+            final var response = testRestTemplate.exchange(
+                "/api/smoketest/offenders/A1234AA/release",
+                HttpMethod.PUT,
+                request,
+                Void.class
+            );
+
+            assertThat(response.getStatusCode()).isEqualTo(FORBIDDEN);
+        }
 
+
+        @Test
+        @DisplayName("not found")
+        public void notFound() {
+
+            final var response = testRestTemplate.exchange(
+                "/api/smoketest/offenders/NOT_AN_OFFENDER/release",
+                HttpMethod.PUT,
+                createHttpEntity(authTokenHelper.getToken(AuthTokenHelper.AuthToken.SMOKE_TEST), null),
+                Void.class
+            );
+
+            assertThat(response.getStatusCode()).isEqualTo(NOT_FOUND);
+        }
+
+        @Test
+        @DisplayName("will release the prisoner")
+        public void willReleasePrisoner() {
+
+            final var response = testRestTemplate.exchange(
+                "/api/smoketest/offenders/A1234AA/release",
+                HttpMethod.PUT,
+                createHttpEntity(authTokenHelper.getToken(AuthTokenHelper.AuthToken.SMOKE_TEST), null),
+                Void.class
+            );
+            assertThat(response.getStatusCode()).isEqualTo(OK);
+
+            // TODO Remove this line - this is just to reset the imprisonment status back to its original
```

Comment:
minor typo on comment "imprisonment status" should be recall

Reply:
yes - will update in next PR